apps/create-plugin/src/index.ts

```diff
@@ -0,0 +1,141 @@
+import { Command } from 'commander'
+import packageJson from '../package.json'
+import { createPlugin, logBoostStars } from './builder'
+const _prompts = require('prompts')
+const figlet = require('figlet')
+// import { Chains } from '@rabbitholegg/questdk-plugin-utils'
```

Comment:
This should work now

Reply:
it does not :/

<img width="1160" alt="Screenshot 2024-02-07 at 6 52 44 PM" src="https://github.com/rabbitholegg/questdk-plugins/assets/7825326/23472596-70bd-43f4-84a4-976d13821644">

Reply:
This looks like a problem with the plugin creator packages build vs an issue with the import. I can fix it if you want but I want to rely on these shared utils where possible now that we have them.

Reply:
I'll agree on reusing em - I tried - and I'll take you up on the fix, I'm not very experienced with typescript builds.

Reply:
we could also resolve this one later it doesn't feel critical